types/mock-agent.d.ts

```diff
@@ -59,6 +59,9 @@ declare namespace MockAgent {
     /** Ignore trailing slashes in the path */
     ignoreTrailingSlash?: boolean;
 
+    /** Accept URLs with search parameters using non standard syntaxes. default false */
+    acceptNonStandardSearchParameters?: boolean;
```

Comment:
Can you add a test for this type? We use tsd.

Reply:
Sorry, what do you mean? are these not enough?

https://github.com/nodejs/undici/blob/716fb9447ec29095215b4e566b1e9120b1b64fd7/test/mock-agent.js#L2892-L2982

Reply:
sure, sorry I missed that 🙂👍

Reply:
Done :+1: 

is this ok? :slightly_smiling_face:

https://github.com/nodejs/undici/pull/4148/commits/cf7daab4e54a5a8b0fffbaba63a0fb02eae9faf1